src/gui/clips/AutomationClipView.cpp

```diff
@@ -185,6 +202,11 @@ void AutomationClipView::constructContextMenu( QMenu * _cm )
 	_cm->addAction( embed::getIconPixmap( "flip_x" ),
 						tr( "Flip Horizontally (Visible)" ),
 						this, SLOT(flipX()));
+	if (AutomatableModel::s_lastChangedModel != nullptr && !AutomatableModel::s_lastChangedModel->displayName().isEmpty())
```

Comment:
Eh, not a big fan of checking if the model name is empty and never adding the option in such a case to be honest. If it somehow turns out to be empty, that probably would mean that there is a bug elsewhere in the code that we should fix.

Reply:
I only added the check because I thought it was be confusing for users to see they can connect a random unnamed model, one which they may not have even directly changed. For example, when a clip is added, two new models appear: "Mute", and "". It may appear odd to users adding a new clip and rightclicking on it to see that they can "Connect last changed model ()"

There are many other unnamed automatable models. It may be outside of the scope of this pr to give them all names, but for now I can remove the check for empty name just so that they function the same way. I suppose if they are visible, it will be easier for other people to track them down and name them eventually.